data/selinux/snappy.te

```diff
@@ -402,11 +402,13 @@ allow snappy_t var_t:sock_file unlink;
 allow snappy_t unconfined_t:dir search;
 allow snappy_t unconfined_t:file { open read };
 
-# Allow snapd to get file attributes in policykit_auth_exec_t domain
+# Allow snapd to get file attributes in policykit{,_auth}_exec_t domain
 gen_require(`
     type policykit_auth_exec_t;
+    type policykit_exec_t;
 ')
 allow snappy_t policykit_auth_exec_t:file { getattr };
```

Comment:
I think snapd may as well poke /usr/lib/policykit-1/polkitd so we'll need this too:
```suggestion
allow snappy_t policykit_auth_exec_t:file { getattr };
allow snappy_t policykit_exec_t:file { getattr };
```
for reference the expected context of polkitd:
https://github.com/SELinuxProject/refpolicy/blob/504feb7a98e2e70f774d6fe7107b5d1a5f2c6124/policy/modules/services/policykit.fc#L9